metadata-ingestion/src/datahub/ingestion/source/dbt.py

```diff
@@ -797,7 +769,7 @@ def get_schema_metadata(
     if node.max_loaded_at is not None:
         actor = mce_builder.make_user_urn("dbt_executor")
         last_modified = AuditStamp(
-            time=int(dateutil.parser.parse(node.max_loaded_at).timestamp() * 1000),
+            time=int(node.max_loaded_at.timestamp() * 1000),
```

Comment:
Can we use https://github.com/datahub-project/datahub/blob/master/metadata-ingestion/src/datahub/utilities/time.py#L13 instead?

Reply:
yep will make the change in https://github.com/datahub-project/datahub/pull/5970